### ISSUE TITLE
Navbar followup issue

### ISSUE THREAD

**Issue by amr-thameen** [empirical-org/Empirical-Core#4022]
Once Emilia is done with the original issues for the nav bar, she will come back to this

- [ ] Change color of hover state to #7cfbdf
- [ ] Hover state for the link stays when you are on that page
- [ ] If you click on a link, it will make the dropdown sticky but clicking anywhere outside should hide it
